src/component/admin/groups/GroupsList/GroupCard/GroupCard.tsx

```diff
@@ -55,30 +71,32 @@ const StyledCounterDescription = styled('span')(({ theme }) => ({
     marginLeft: theme.spacing(1),
 }));
 
-const ProjectBadgeContainer = styled('div')(() => ({}));
+const ProjectBadgeContainer = styled('div')(() => ({
+    width: '50%',
```

Comment:
If we're going with this approach, I think we'll also need this:
```suggestion
    display: 'flex',
    justifyContent: 'end',
    width: '50%',
```
Otherwise the project badges get misaligned:
![image](https://user-images.githubusercontent.com/14320932/182594385-8dd169cc-8b21-4411-b958-a6a96ffc63a1.png)

Reply:
I went with maxWidth and that should cover it?